pkg/migrate/migrate.go

```diff
@@ -1146,3 +1296,249 @@ func resetReclaimPolicy(ctx context.Context, w *log.Logger, clientset k8sclient.
 
 	return nil
 }
+
+// buildPVConsumerPod creates a pod spec for consuming a pvc
+func (p *PVMigrator) buildPVConsumerPod(pvcName string) *corev1.Pod {
+	podName := p.podNameOverride
+	if podName == "" {
+		tmp := uuid.New().String()[:5]
+		podName = fmt.Sprintf("pvmigrate-vol-consumer-%s-%s", pvcName, tmp)
+	}
+	if len(podName) > 63 {
+		podName = podName[0:31] + podName[len(podName)-32:]
+	}
+	return &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      podName,
+			Namespace: "default",
```

Comment:
should this be in the same namespace as the pvc?

Reply:
We make a copy of the src PVC in the `default` namespace so that's why the pod is created there.

Reply:
The copy needs to be in the same ns as the source PVC.

Yes, really.